compose/completion.md

```diff
@@ -13,77 +13,84 @@ for the bash and zsh shell.
 
 Make sure bash completion is installed.
 
-*  On a current Linux OS (in a non-minimal installation), bash completion should be
+#### Linux
+
+1. On a current Linux OS (in a non-minimal installation), bash completion should be
 available.
+2. Place the completion script in `/etc/bash_completion.d/`.
 
-*  On a Mac, install with `brew install bash-completion`.
+```shell
+sudo curl -L https://raw.githubusercontent.com/docker/compose/{{site.compose_version}}/contrib/completion/bash/docker-compose -o /etc/bash_completion.d/docker-compose
+```
 
-Place the completion script in `/etc/bash_completion.d/`
-(or `/usr/local/etc/bash_completion.d/` on a Mac):
+### Mac
+
+##### Install via Homebrew
+
+1. Install with `brew install bash-completion`.
+2. Place the completion script in `/usr/local/etc/bash_completion.d/`.
```

Comment:
I think step `2` is describing the `curl` command below. 

There seems to be an inconsistency with what step 2 describes and what the command does. 
Step `2` says to place the completion script in `/usr/local/etc/bash_completion.d/` but the `curl` command has 

`-o /etc/bash_completition.d/docker-compose`. 

So it isn't clear what is intended.

Reply:
Hi @jose-bigio, we have resolved the inconsistencies you have pointed out by testing the procedure on Mac.